code/modules/mob/living/carbon/human/species.dm

```diff
@@ -815,7 +815,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		H.losebreath = 0
 
 		var/takes_crit_damage = (!HAS_TRAIT(H, TRAIT_NOCRITDAMAGE))
-		if((H.health < H.crit_threshold) && takes_crit_damage)
+		if((H.health <= H.crit_threshold) && takes_crit_damage && name != "IPC")
```

Comment:
String comparisons are expensive.

Reply:
Also couldn't you just add `TRAIT_NOCRITDAMAGE`?

Reply:
Yeah I guess I could. I'll do that tomorrow.